set-start.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+var path = require('path')
+var fs = require('fs')
+
+var save = process.env.npm_config_save
+
+if (save) {
+  var fileName = path.join('..', '..', 'package.json')
```

Comment:
Yes, thanks a lot @Haroenv for reviewing the pull request 👍 Can you please change that @klazich?